package.json

```diff
@@ -167,32 +167,32 @@
     "react-dom": "^18.2.0",
     "react-image-crop": "^11.0.7",
     "react-keyed-flatten-children": "^3.0.0",
-    "react-native": "0.74.1",
-    "react-native-compressor": "^1.8.24",
-    "react-native-date-picker": "^4.4.2",
-    "react-native-drawer-layout": "^4.0.1",
-    "react-native-gesture-handler": "2.20.0",
+    "react-native": "0.76.3",
+    "react-native-compressor": "1.9.0",
+    "react-native-date-picker": "^5.0.7",
+    "react-native-drawer-layout": "^4.0.3",
+    "react-native-gesture-handler": "2.20.2",
     "react-native-get-random-values": "~1.11.0",
-    "react-native-image-crop-picker": "0.41.2",
+    "react-native-image-crop-picker": "0.41.6",
     "react-native-ios-context-menu": "^1.15.3",
-    "react-native-keyboard-controller": "^1.14.0",
+    "react-native-keyboard-controller": "^1.14.5",
     "react-native-mmkv": "^2.12.2",
-    "react-native-pager-view": "6.2.3",
-    "react-native-picker-select": "^9.1.3",
+    "react-native-pager-view": "6.6.0",
+    "react-native-picker-select": "^9.3.1",
     "react-native-progress": "bluesky-social/react-native-progress",
-    "react-native-qrcode-styled": "^0.3.1",
-    "react-native-reanimated": "^3.16.1",
+    "react-native-qrcode-styled": "^0.3.3",
+    "react-native-reanimated": "^3.16.3",
     "react-native-root-siblings": "^4.1.1",
-    "react-native-safe-area-context": "4.10.1",
-    "react-native-screens": "~3.31.1",
-    "react-native-svg": "^15.3.0",
-    "react-native-uitextview": "^1.3.0",
+    "react-native-safe-area-context": "4.14.0",
+    "react-native-screens": "~4.3.0",
```

Comment:
Hmm that is quite interesting that bumping to `react-native-screens` v4 works correctly for you if you use `native-stack`.  You are not using `@react-navigation` v7 and it is not recommended to bump one without the other: https://github.com/software-mansion/react-native-screens#usage-with-react-navigation.  But I mean, if it works, then 🤷‍♂️ 

What's more, `react-native-screens` v4 come with our own implementation of bottom-sheets connected to navigation: https://blog.swmansion.com/introducing-react-native-screens-4-0-0-1b833ff98a55. Maybe it would be good to research it too one day if you are unhappy with current bottom-sheet 😅

Reply:
Huh, yea this was actually a recommendation from the author!

https://bsky.app/profile/satya164.bsky.social/post/3lcawiutecs2g

Mostly I was just testing to see if it would work (and was sorta surprised it did!) Do you think it'd be best to just change this back?

Reply:
Also: https://bsky.app/profile/satya164.bsky.social/post/3lcaxcltl4m2j

Reply:
> What's more, react-native-screens v4 come with our own implementation of bottom-sheets connected to navigation: [blog.swmansion.com/introducing-react-native-screens-4-0-0-1b833ff98a55](https://blog.swmansion.com/introducing-react-native-screens-4-0-0-1b833ff98a55). Maybe it would be good to research it too one day if you are unhappy with current bottom-sheet 😅

Now _this_ is something i'd definitely be interested in trying!!

Reply:
i got wrecked by types there, would need some work to figure out what's going on

Reply:
> Huh, yea this was actually a recommendation from the author!

Oh so probably `react-navigation` fixed all the problems with integration, nice!

Reply:
> Mostly I was just testing to see if it would work (and was sorta surprised it did!) Do you think it'd be best to just change this back?

I mean, if it works, then I think it's fine! I would just suggest bumping the navigation to v7 ASAP too. Then you'll be sure everything is up to date and supported.